Configurateur/front/lib/screens/company-profil/object-creation.dart

```diff
@@ -25,6 +25,7 @@ import 'package:http/http.dart' as http;
 import 'package:http_parser/http_parser.dart';
 import 'package:image_picker/image_picker.dart';
 import 'package:provider/provider.dart';
+import 'package:front/components/custom_toast.dart';
```

Comment:
why add this when it's not used in the page

Reply:
because it's used.